client/app/components/composites/Avatar/ArrowDropdown.css

```diff
@@ -0,0 +1,66 @@
+.arrowDropdown {
+}
+
+.rootArrow {
+  width: 0;
+  height: 0;
+  border-left: 8px solid transparent;
+  border-right: 8px solid transparent;
+  border-bottom: 8px solid var(--topbarBorderColor);
+}
+
+.box {
+  position: absolute;
+  border: 1px solid var(--topbarBorderColor);
+  right: 18px; /* we need a var here, whatever our generic margin is*/
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  /* Notifications_box: */
+  background: #FFFFFF;
+  box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.10);
+  border-radius: 3px;
+  z-index: 10;
```

Comment:
This comes from clash between old search bar below. (Two absolutely positioned containers in parallel). 'z-index: 1' should be enough.